editor/editor_export.cpp

```diff
@@ -1301,6 +1305,8 @@ Error EditorExportPlatform::save_pack(const Ref<EditorExportPreset> &p_preset, b
 		f->store_buffer(buf, got);
 	}
 
+	ftmp = Ref<FileAccess>(); // Close temp file.
```

Comment:
Wouldn't it be clearer to use `close()` for this?

Reply:
Right, that makes sense. It just looks a bit weird to have to assign a new empty ref to force closing and destroying an existing FileAccess.

I guess another is memdelete but that's not really better.

Reply:
Yes that sounds good!